interfaces/cython/cantera/examples/onedim/diffusion_flame_extinction.py

```diff
@@ -113,11 +113,11 @@
     f.set_profile('lambda', normalized_grid, f.L * strain_factor ** exp_lam_a)
     try:
         f.solve(loglevel=0)
-    except Exception as e:
-        # Throw Exception if solution fails
+    except ct.CanteraError as e:
+        # Throw Error if solution fails
```

Comment:
I think you can probably just delete the comment, it doesn't add anything anyways.